package.json

```diff
@@ -13,14 +13,15 @@
     "flow": "flow",
     "flow-typed": "rimraf flow-typed/npm && flow-typed install --overwrite || true",
     "lint": "cross-env NODE_ENV=development eslint --cache --format=node_modules/eslint-formatter-pretty .",
-    "lint-fix": "npm run lint -- --fix",
+    "lint-fix": "npm run lint -- --fix && prettier --single-quote --write '**/*.js'",
```

Comment:
We're linting `**/*.js` but files in the `**/**/*.js` glob are linted as well. Hmm interesting. For example, `app/components/*.js` files are listed. Just wondering why this is

Reply:
Prettier use [glob syntax from the glob module](https://github.com/isaacs/node-glob/blob/master/README.md#glob-primer). 
`**` matches zero or more directories and subdirectories.

> The following characters have special magic meaning when used in a path portion:
> * ...
> * ** If a "globstar" is alone in a path portion, then it matches zero or more directories and subdirectories searching for matches. It does not crawl symlinked directories.

Reply:
Maybe put the prettier before eslint --fix is better? What do you think?

Reply:
Well if eslint auto fixes through the prettier eslint config then shouldn't we just need to eslint auto fix?

Reply:
I think `eslint --fix` and prettier have some different. There is a example show the different in this [discuss](https://github.com/prettier/prettier-eslint/issues/101).

Now I think the original way is better.  Use `eslint --fix` first and then use prettier to format fixed code. Please forgive my capricious personality.

```js
"lint-fix": "npm run lint -- --fix && prettier --single-quote --write '**/*.js'",
```